File: Learn/Lang-CPP/A000-HelloWorld/HelloWorld.cpp
#include <HelloWorld.hpp>

//using namespace std;

const std::string str_helloWorld = "Hello World from me !! - Variable String";
const char sz_char_helloWorld[] = "Hello World from me !! - Char Array String";

int main(){        
    std::cout << "Hello World from me !! - Inline String" << std::endl;
    std::cout << str_helloWorld << std::endl;
    std::cout << Get_strHelloWorld() << std::endl; //Call from header
    return 0;
}

File: Learn/Lang-Dart/A000-HelloWorld/HelloWorld.dart
void main() {
  print("Hello World !! - Main");
  print(K_HELLOWORLD);
  print(str_helloWorld);
  Do_HelloWorld();
  Do_HelloWorldInline();
  print(Get_HelloWorld());
}

const String K_HELLOWORLD = "Hello World !! - Const Variable";
String str_helloWorld = "Hello World !! - Variable";
String Get_HelloWorld() {
  return "Hello World !! - Get Function";
}
void Do_HelloWorld() => print("Hello World !! - Function");
void Do_HelloWorldInline({String param_helloWorld = "Hello World !! - Function Param"}) { 
  print(param_helloWorld);
}